packages/web-app-files/src/helpers/share/role.ts

```diff
@@ -193,6 +207,41 @@ export const linkRoleUploaderFolder = new LinkShareRole(
   $gettext('uploader'),
   [SharePermissions.create]
 )
+export const spaceViewer = new SpaceShareRole(
+  'spaceViewer',
+  false,
+  $gettext('Viewer'),
+  $gettext('viewer'),
+  [SharePermissions.read]
+)
+export const spaceEditor = new SpaceShareRole(
```

Comment:
Please call it `spaceRoleEditor` instead (other roles also have a `Role` part in the name, would like to keep it consistent)